lib/http.js

```diff
@@ -232,6 +232,14 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       const {profileAgent} = await profileAgents.get({id: profileAgentId});
 
       profileAgent.sequence++;
+      const now = Date.now();
+      const ttl = config['profile-http'].zcap.default.ttl;
+      for(const props in zcaps) {
```

Comment:
this is fine, but I'm not sure this is plural. `props` implies multiple properties here we have a single `prop` aka property from the zcaps object. So I would use `prop` or `property` here. This isn't a big deal and if time is a concern you can ignore this comment.

Reply:
yeah, this makes more sense. I can update it.